### ISSUE TITLE
pyodide libraries

### ISSUE THREAD

**Issue by ifuchs** [holzschu/a-shell#373]
i presume that ashell could run a python webserver which could deliver the pyodide library modules but can they run in the ios browser? Has anyone tried?

**Comment by personalizedrefrigerator** [comment]
Pyodide works in Safari for iOS. It can be tried here: https://pyodide.org/en/stable/console.html

**Comment by ifuchs** [comment]
@personalizedrefrigerator Yes, the console runs fine. However, scripts such as this: https://www.itp.uni-hannover.de/en/groups/weimer/teaching/ideal-gas-simulator/ do not seem to work on ios but it may be due to the version they are using.
